packages/summary-list/src/SummaryList.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Text } from '@ag.ds-next/text';
+import { TextLink } from '@ag.ds-next/text-link';
+import { Stack } from '@ag.ds-next/box';
+import {
+	SummaryList,
+	SummaryListItem,
+	SummaryListItemTerm,
+	SummaryListItemDescription,
+	SummaryListItemAction,
+} from './SummaryList';
+
+export default {
+	title: 'Content/SummaryList',
+	component: SummaryList,
+} as ComponentMeta<typeof SummaryList>;
+
+export const Basic: ComponentStory<typeof SummaryList> = (args) => (
+	<SummaryList {...args}>
```

Comment:
```suggestion
export const Basic = () => (
	<SummaryList>
```

Summary list doesn't have any props that can be configured (only children), so I think we should remove this.